public/app/plugins/datasource/prometheus/components/VariableQueryEditor.tsx

```diff
@@ -133,61 +133,67 @@ export const PromVariableQueryEditor = ({ onChange, query, datasource }: Props)
+          labelWidth={20}
+          tooltip={
+            <div>The Prometheus data source plugin provides the following query types for template variables.</div>
+          }
+        >
+          <Select
+            placeholder="Select query type"
+            aria-label="Query type"
+            onChange={onQueryTypeChange}
+            onBlur={handleBlur}
+            value={qryType}
+            options={variableOptions}
+            width={25}
+          />
+        </InlineField>
+      </InlineFieldRow>
       {qryType === QueryType.LabelValues && (
         <>
-          <InlineField
-            label="Label"
-            labelWidth={20}
-            required
-            tooltip={
-              <div>
-                Returns a list of label values for the label name in all metrics unless the metric is specified.
-              </div>
-            }
-          >
-            <Select
-              aria-label="label-select"
-              onChange={onLabelChange}
-              onBlur={handleBlur}
-              value={label}
-              options={labelOptions}
-              width={25}
-              allowCustomValue
-            />
-          </InlineField>
-          <InlineField
-            label="Metric"
-            labelWidth={20}
-            tooltip={<div>Optional: returns a list of label values for the label name in the specified metric.</div>}
-          >
-            <Input
-              type="text"
-              aria-label="Metric selector"
-              placeholder="Optional metric selector"
-              value={metric}
-              onChange={onMetricChange}
-              onBlur={handleBlur}
-              width={25}
-            />
-          </InlineField>
+          <InlineFieldRow>
+            <InlineField
+              label="Label"
+              labelWidth={20}
+              required
+              tooltip={
+                <div>
+                  Returns a list of label values for the label name in all metrics unless the metric is specified.
+                </div>
+              }
+            >
+              <Select
+                aria-label="label-select"
+                onChange={onLabelChange}
+                onBlur={handleBlur}
+                value={label}
+                options={labelOptions}
+                width={25}
+                allowCustomValue
+              />
+            </InlineField>
+          </InlineFieldRow>
+          <InlineFieldRow>
+            <InlineField
+              label="Metric"
+              labelWidth={20}
+              tooltip={<div>Optional: returns a list of label values for the label name in the specified metric.</div>}
+            >
+              <Input
+                type="text"
+                aria-label="Metric selector"
+                placeholder="Optional metric selector"
+                value={metric}
+                onChange={onMetricChange}
+                onBlur={handleBlur}
+                width={100}
+              />
```

Comment:
Why are we not using the Prometheus MetricSelect component here that has advanced search and lookup and instead have users have to remember and write it manually, or open another tab & query editor to find a metric name?

Reply:
https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/querybuilder/components/MetricSelect.tsx

Reply:
This is a good suggestion! This would be a better solution than what is implemented. Originally we were looking at having the same functionality as the string input where users simply input `label_values(<some_label>, <some_metric>)`. There is room for improvement here, should I replace the metric input with the metric select component here or in another PR? @catherineymgui If we replace the input with the metric select, we won;t have to worry about the length of the input. WDYT?

Reply:
Yeah, I was questioning this but didn't know if we wanted to just focus on the size of fields

Select makes more sense to me, but it currently wraps to the length of the metric right? To me it makes more sense to still use a fixed width as this behaviour isn't really on the rest of the form. Do we want to just update this to the new field component and just match the width with the rest of the page?